builtins/amp-img.md

```diff
@@ -109,12 +109,14 @@ This attribute is similar to the `src` attribute on the `img` tag. The value mus
 
 Same as `srcset` attribute on the `img` tag. For browsers that do not support `srcset`, `<amp-img>` will default to using `src`. If only `srcset` and no `src` is provided, the first url in the `srcset` will be selected.
 
-**sizes**
+**sizes (optional)**
 
-Same as `sizes` attribute on the `img` tag.
+For the `<img>` tag in `HTML`, the `sizes` attribute is used in conjunction with the `srcset` attribute and specifies the intended display size of the image as corresponding to a media condition. It will influence the intrinsic display size of the underlying `<img>`. If provided, the `sizes` attribute given to `<amp-img>` is propagated to the underlying `<img>` tag. However, `sizes` in AMP has the additional effect of setting inline `width` on the `<amp-img>` in AMP according to the matched media query. If the `srcset` attribute is provided, `<amp-img>` will autogenerate a `sizes` attribute for the underlying `<img>` when none is specified.
```

Comment:
Let's add a section above this paragraph talking about "you don't really need to set this, AMP does auto sizes if you just provide srcset, this is awesome, etc.. etc..." and then say if you really want to set `sizes`, be aware of the caveats below.. right now the auto size feature is sort of buried at the end.

Reply:
+1 on rearranging the order of things. I would start with the benefit of using this attribute (why its an option), followed by the default behavior if it isn't used.

I'm a bit confused about how this works with media queries - I'm able to set defined widths for the image depending on the screen size with this attribute?

Reply:
Let me clarify what I'm trying to do here: 
1. We just added a feature that makes people no longer need this attribute, so really most people should no longer need this attribute at all. 
2. My original writing was more along the lines of: 
a) This attribute does X. 
b) We have auto-sizes so for the most part you don't need this. 
c) If you really want to use `sizes` beware of caveats. 

The slightly weird thing that I also tried to do is I think most people don't really understand what `srcset` and `sizes` do for vanilla `<img>` in general, so I tried to also explain that, but maybe that's confusing. I copy-pasta-ed this into a [doc](https://docs.google.com/document/d/1uXxLhTHaWn3OTCq6GjkPPsWhQGsmNM1FVE3ozwBR77M/edit?hl=en) for easier commenting/editing.

Reply:
Thanks Cathy - I've added some suggestions to the doc, the largest one being to change the `read` flag to wrap it around the information on `<img>` tag behavior rather than the guide link.

Reply:
I've pushed the changes. PTAL =)